ConfusedPolarBear.Plugin.IntroSkipper/Configuration/inject.ts

```diff
@@ -1,57 +1,111 @@
-const introSkipper = {
-    originalFetch: window.fetch.bind(window),
-    d: msg => console.debug("[intro skipper] ", msg),
+interface ApiClient {
+    serverAddress(val?: string): string;
+    accessToken(): string;
+}
+
+declare const ApiClient: ApiClient;
+
+interface SkipperData {
+    Introduction?: Segement
+    Credits?: Segement
+}
+
+interface Segement {
```

Comment:
Is this normal ?
```suggestion
interface Segment {
```
It should be like this right ?